packages/test/src/test-workflows.ts

```diff
@@ -22,6 +22,7 @@ import { parseWorkflowCode } from '@temporalio/worker/lib/worker';
 import * as activityFunctions from './activities';
 import { cleanStackTrace, REUSE_V8_CONTEXT, u8 } from './helpers';
 import { ProcessedSignal } from './workflows';
+import VersioningIntent = coresdk.common.VersioningIntent;
```

Comment:
TIL about this `import ... = something other than require` syntax. Can't even find any documentation about it. Nice to know.

Reply:
I think this is an old way of importing, I would rather avoid this style for consistency.

Reply:
Didn't even realize. I think the IDE did it automatically